packages/myst-common/src/plural.ts

```diff
@@ -15,8 +15,8 @@ export function plural(f: string, count?: number | any[] | Record<any, any>): st
     (typeof count === 'number'
       ? count
       : Array.isArray(count)
-        ? count?.length
-        : Object.keys(count ?? {}).length) ?? 0;
+      ? count?.length
+      : Object.keys(count ?? {}).length) ?? 0;
   return f
```

Comment:
this change looks sensible to fix the linter, local `npm run lint` is happy, yet ci still fails -- @fwkoch @agoose77 has anything changed around our prettier config recently?